spec/libraries/duplicator_spec.rb

```diff
@@ -0,0 +1,267 @@
+require 'rails_helper'
+
+RSpec.describe Duplicator do
+  class SimpleObject
+    def initialize(id)
+      @id = id
+    end
+
+    def duplicate(duplicator)
```

Comment:
Unused method argument - `duplicator`. If it's necessary, use `_` or `_duplicator` as an argument name to indicate that it won't be used. You can also write as `duplicate(*)` if you want the method to accept any arguments but don't care about them.